src/libs/protocol-data-provider/index.tsx

```diff
@@ -8,10 +8,18 @@ import {
   getProvider,
   CustomMarket,
 } from '../../helpers/config/markets-and-network-config';
+import { aggregatorV3InterfaceABI, TokenPrice } from '../../ui-config/markets';
+import { BigNumber } from '@aave/protocol-js';
+import { useEffect } from 'react';
+import { initCornerstoneSDKWithSigner } from '@corndao/corn-sdk';
+import { Near } from 'near-api-js';
+import { IncentivesController } from '@pret/contract-helpers';
 
 const LS_KEY = 'selectedMarket';
 
 export interface ProtocolContextData {
+  showMarketTableItemAPYCell: boolean;
```

Comment:
These are not protocol context data I think. Maybe create another data provider for prices?

Reply:
sure. extracted .